app/serializers/v1/coupon_serializer.rb

```diff
@@ -15,7 +15,7 @@ def serialize
         frequency_duration: model.frequency_duration,
         created_at: model.created_at.iso8601,
         expiration: model.expiration,
-        expiration_duration: model.expiration_duration
+        expiration_date: model.expiration_date&.iso8601
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.